src/sign-in-webapp/README.md

```diff
@@ -67,46 +70,143 @@ Later the web app can make calls to the Microsoft 365 Graph API on behalf of sig
-   ```
+<details>
+   <summary>:computer: Alternative: modify the `appsettings.json` file from your terminal</summary>
 
-   :warning: Today, with ASP.NET Core 3.1 without Microsoft Identity Web, when running `dotnet new --auth` and creating a `webapp` from an ASP.NET core template, the application is targeting the Azure AD v1.0 endpoint. Microsoft Identity Web provides the glue between the ASP.NET Core middleware and MSAL .NET to bring a clearer, more robust developer experience, which also leverages the power of the Microsoft identity platform (formerly Azure AD v2.0 endpoint). `Microsoft.Identity.Web` is a simpler way to use Azure AD in ASP.NET Core web apps and web APIs. It doesn't replace ASP.NET Identity in any way, it doesn't replace AddOpenIdConnect, AddJwtBearer or AddCookie or any of the lower level primitives, but it does use and configure them correctly for Azure AD. It replaces AzureAD.UI and AzureADB2C.UI which are obsolete in .NET 5.0 or later. For more information, please take a look at https://github.com/AzureAD/microsoft-identity-web/wiki/Microsoft-Identity-Web-basics#why-use-microsoftidentityweb
+1. Create the `appsettings.json` file with the Azure AD app configuration
 
-## Configure the web app
+   ```bash
+   cat > appsettings.json <<EOF
+   {
+     "AzureAd": {
+       "Instance": "https://login.microsoftonline.com/",
+       "Domain": "${AZURE_AD_APP_DOMAIN}",
+       "ClientId": "${AZURE_AD_APP_CLIENT_ID_WEBAPP}",
+       "TenantId": "$(az account show --query tenantId --output tsv)",
+       "ClientSecret": "[Copy the client secret added to the app from the Azure portal]",
+       "ClientCertificates": [
+       ],
+       "CallbackPath": "/signin-oidc"
+     },
+     "DownstreamApi": {
+       "BaseUrl": "https://graph.microsoft.com/v1.0/me",
+       "Scopes": "user.read"
+     },
+     "Logging": {
+       "LogLevel": {
+         "Default": "Information",
+         "Microsoft.AspNetCore": "Warning"
+       }
+     },
+     "AllowedHosts": "*"
+   }
+   EOF
+   ```
 
 1. Set the client secret app
 
    ```bash
    export AzureAd__ClientSecret=$AZURE_AD_APP_SECRET
    ```
+</details>
+
+## Run the application
 
-## Run the web API
+### 1. Run the webapp
 
 1. Execute the following command to get the app up and running:
 
    ```bash
    dotnet run
    ```
 
-## Signin into the web app
+### 2. Signin into the web app
 
 1. Once the web app is listening, navigate to https://localhost:5001
+1. Sign-in with your user credentials.
 
-## Clean up
+### 3. Clean up
 
 1. Delete the Azure AD app
 
    ```bash
    az ad app delete --id $AZURE_AD_APP_CLIENT_ID_WEBAPP
    ```
+
+## About the code
+
+The ASP.NET Core 6.0 Web App will allow users to sign-in, so it can retrieve a Security Token scoped specifically for the Microsoft Graph API, and will use that token to access the user's information. For more information about the proposed scenario, please take a look at the following diagram:
+
+```output
```

Comment:
Are these ASCII diagrams hard to build? I'm not sure we're going to use them (we might switch to images of diagrams) and would rather you not do any "throwaway" work if it's a hassle. I like them, but we likely won't use them as-is.

Reply:
they're meant to replaced by images. They mainly prototypes for content dev team create diagrams on top of them. 

Any desire to remove them before content dev team create the actual `visio`/`draw.io` diagrams? Open to that.

Reply:
Let's go ahead and just remove the ascii.

Reply:
done | addressed from [59505](https://github.com/Azure-Samples/ms-identity-docs-code-dotnet/pull/14/commits/59505ca70b163e44371c02bef614150860ad3600)

Reply:
Indeed, it's gone.